reactjs/src/components/Styles.js

```diff
@@ -1,3 +0,0 @@
-export const rightMargin = {
```

Comment:
Delete this file as its not in use now

Reply:
This file is already deleted. 

Please check commit https://github.com/prakashsejwani/todo-frameworks/pull/7/commits/0d451e884f100ab88d7cc3bc610c724604f910bf